packages/trimerge-sync/src/merge-heads.ts

```diff
@@ -30,34 +31,36 @@ export function mergeHeads<N extends CommitInfo>(
   getCommit: GetCommitFn<N>,
   merge: MergeCommitsFn,
 ): string | undefined {
-  headRefs.sort(sortRefs);
-  const visitors = headRefs.map(
-    (ref): Visitor => ({
+  function sortFn(a: Visitor, b: Visitor): number {
```

Comment:
nit: `sortVisitors`? even in this small scope, I'm finding myself getting sortRefs and sortFn confused.